distributed/cfexecutor.py

```diff
@@ -0,0 +1,165 @@
+from __future__ import print_function, division, absolute_import
+
+import weakref
+
+import six
+
+from toolz import merge
+
+from tornado import gen
+
+from concurrent.futures import Future, Executor, TimeoutError
```

Comment:
Given that these terms are used elsewhere in the codebase to mean different things it might be wise to instead use the namespace explicitly.  Perhaps the following?

```python
import concurrent.futures as cf

cf.Future, cf.Executor, cf.TimeoutError
```